main.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  aws_region = "us-east-1"
+  env        = "test"
+}
+
+provider "aws" {
+  region = local.aws_region
+}
+
+data "aws_eks_cluster" "eks" {
+  name = module.eks.cluster_id
+}
+
+data "aws_eks_cluster_auth" "eks" {
+  name = module.eks.cluster_id
+}
+
+provider "kubernetes" {
+  host                   = data.aws_eks_cluster.eks.endpoint
+  cluster_ca_certificate = base64decode(data.aws_eks_cluster.eks.certificate_authority[0].data)
+  token                  = data.aws_eks_cluster_auth.eks.token
+}
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "3.2.0"
+
+  azs             = ["us-east-1a", "us-east-1b"]
+  name            = "main"
```

Comment:
Naming should be consistent between VPC and EKS 
Use ${local.env} here as well

Reply:
changed.